src/Microsoft.Health.Dicom.SqlServer/Features/Schema/Sql/Sprocs/DeleteInstanceV6.sql

```diff
@@ -254,9 +258,9 @@ BEGIN
     END
 
     INSERT INTO dbo.ChangeFeed
-    (Action, PartitionKey, StudyInstanceUid, SeriesInstanceUid, SopInstanceUid, OriginalWatermark)
-    SELECT 1, PartitionKey, StudyInstanceUid, SeriesInstanceUid, SopInstanceUid, Watermark
-    FROM @deletedInstances
+    (Action, PartitionKey, StudyInstanceUid, SeriesInstanceUid, SopInstanceUid, OriginalWatermark, FilePath)
```

Comment:
Do we need to store the FilePath for deleted actions? Trying to understand how long these files will exist before they can even access it. 
Also can we consistent with the alias. Can we use lowe case no caps